timesketch/api/v1/resources/explore.py

```diff
@@ -156,15 +156,26 @@ def post(self, sketch_id: int):
 
         all_indices = list({t.searchindex.index_name for t in sketch.timelines})
         indices = query_filter.get("indices", all_indices)
+        all_timeline_ids = [t.id for t in sketch.timelines]
```

Comment:
Would `sketch.timelines` return also timelines that have been marked deleted?

Reply:
no, they are not part of the timelines lookup, there is no "marked as deleted" thing, they are either there or not.